Source/PickerConfiguration.swift

```diff
@@ -1,6 +1,13 @@
 import UIKit
 
 public class PickerConfiguration {
```

Comment:
Right, but I don't want to conflict with Lightbox and stuff like that, what do you think?